arwen/host/execution.go

```diff
@@ -198,6 +204,10 @@ func (host *vmHost) doRunSmartContractCall(input *vmcommon.ContractCallInput) (v
 }
 
 func copyTxHashesFromContext(runtime arwen.RuntimeContext, input *vmcommon.ContractCallInput) {
+	// for DirectCalls tx hashses are not filled
+	if input.CallType != vm.DirectCall {
```

Comment:
condition does not match the comment - or I am missing something.

Reply:
done